src/packages/solid/components/Tile/Tile.tsx

```diff
@@ -143,9 +72,11 @@ const Tile: Component<TileProps> = (props: TileProps) => {
 
         <View
           x={
-            (props.width ??
-              styles.Container.tones[props.tone ?? styles.tone]?.width ??
-              styles.Container.base.width) -
+            Number(
+              props.width ??
+                styles.Container.tones[props.tone ?? styles.tone]?.width ??
+                styles.Container.base.width
```

Comment:
unrelated to this ticket, but while we're here can we memoize these types of calculations to clean up the markup?